sensor/tests/helper/helper.go

```diff
@@ -1,19 +1,22 @@
-package resource
```

Comment:
Did you have maybe some other ideas for that name on your shortlist? 😅

Reply:
Unfortunately no. But this whole file is pending a refactor. @lvalerom were talking about breaking this down into smaller components of the test harness. Like managing sensor instance, interacting with the clusters and asserting eventual resources sent to Central are three examples of responsibilities that are bundled in the same file. 

I'd defer choosing a good name for this package to when we break it down in smaller pieces.